GcmEndpoints/README.md

```diff
@@ -19,7 +21,7 @@ Then choose "App Engine Backend with Google Cloud Messaging".
 
 ![App Engine GCM Module](/doc/img/gcm2.png)
 
-Enter the module/package names for your new backend, and choose the "client" module in your project which contains your Android app. The client module will be set up to call your newly generated backend. Module name which you've entered above (marked with red **1**) will be used in your Android Studio project. Package name (marked with red **2**) will be used for all classes imported from this template and (reversed) for [Endpoints API namespaces](https://developers.google.com/appengine/docs/java/endpoints/annotations#apinamespace). In turn, Endpoints API namespaces will be used to derive the name of the autogenerated Android client libraries, hence this ensures that the names of generated client libraries will match your package name.
+Enter the module/package names for your new backend, and choose the "client" module in your project which contains your Android app. The client module will be set up to call your newly generated backend. The module name which you've entered above (marked with red **1**) will be used in your Android Studio project. The package name (marked with red **2**) will be used for all classes imported from this template and (reversed) for [Endpoints API namespaces](https://developers.google.com/appengine/docs/java/endpoints/annotations#apinamespace). In turn, Endpoints API namespaces will be used to derive the name of the autogenerated Android client libraries, hence this ensures that the names of generated client libraries will match your package name.
```

Comment:
the project which contains your Android app. 
", hence this ensures that" --> ". This ensures that" 
Side note for the future : it's a little easier to review if the lines hard wrap  every 80 chars or so, instead of wrapping; no need to do it here